lib/spack/spack/filesystem_view.py

```diff
@@ -98,6 +98,29 @@ def view_copy(src, dst, view, spec=None):
             )
 
 
+def view_func_parser(parsed_name):
+    # What method are we using for this view
+    if parsed_name in ("hardlink", "hard"):
+        return view_hardlink
+    elif parsed_name in ("copy", "relocate"):
+        return view_copy
+    elif parsed_name == "symlink":
```

Comment:
```suggestion
    elif parsed_name in ("add", "symlink", "soft"):
```

This should match the options from the command:
```
$ spack view -h
usage: spack view [-hv] [-e EXCLUDE] [-d {true,false,yes,no}] ACTION ...

project packages to a compact naming scheme on the filesystem.

positional arguments:
  ACTION
    symlink (add, soft)
                        add package files to a filesystem view via symbolic links
    hardlink (hard)     add packages files to a filesystem view via hard links
    copy (relocate)     add package files to a filesystem view via copy/relocate
    remove (rm)         remove packages from a filesystem view
    statlink (status, check)
                        check status of packages in a filesystem view

optional arguments:
  -h, --help            show this help message and exit
  -v, --verbose         If not verbose only warnings/errors will be printed.
  -e EXCLUDE, --exclude EXCLUDE
                        exclude packages with names matching the given regex pattern
  -d {true,false,yes,no}, --dependencies {true,false,yes,no}
                        Link/remove/list dependencies.
```

Reply:
must we have so many aliases?  Like "add"  Does anyone use add?

Reply:
Seems like we could deprecate some of these.